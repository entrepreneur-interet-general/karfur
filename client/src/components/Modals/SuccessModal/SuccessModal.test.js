import React from 'react';
import { expect } from 'chai'
import { shallow } from 'enzyme'
import SuccessModal from './SuccessModal';


describe('<SuccessModal />', () => {
  it('mounts without crashing', () => {
    const wrapper = shallow(<SuccessModal />);
    wrapper.unmount()
  });

  it('should match its reference snapshot', () => {
    const wrapper = shallow(<SuccessModal />)
  
    expect(wrapper).to.matchSnapshot()
  })
})