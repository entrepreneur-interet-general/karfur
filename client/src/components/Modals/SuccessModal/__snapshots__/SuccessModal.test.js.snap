// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuccessModal /> should match its reference snapshot 1`] = `
<Modal
  classe="success-modal text-center"
>
  <div
    id="svgContainer"
  />
  <h1>
    Succès !
  </h1>
  <p>
    Nous avons enregistré votre article avec succès.
  </p>
  <span
    className="change"
  >
    Cliquez sur le bouton ci-dessous pour le voir en contexte
  </span>
  <Row
    className="align-items-center"
    tag="div"
  >
    <Col
      className="mb-3 mb-xl-0"
      col="6"
      md="2"
      sm="4"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
      xl={true}
    >
      <Link
        replace={false}
        to="/article/undefined"
      >
        <Button
          block={true}
          className="btn-pill"
          color="success"
          outline={true}
          tag="button"
        >
          Ok
        </Button>
      </Link>
    </Col>
  </Row>
</Modal>
`;
